cmd/provider-list/converter.go

```diff
@@ -20,10 +20,20 @@ import (
 	"fmt"
 	"strings"
 
+	xpmetav1 "github.com/crossplane/crossplane/apis/pkg/meta/v1"
+	xpmetav1alpha1 "github.com/crossplane/crossplane/apis/pkg/meta/v1alpha1"
+	xppkgv1 "github.com/crossplane/crossplane/apis/pkg/v1"
+	xppkgv1beta1 "github.com/crossplane/crossplane/apis/pkg/v1beta1"
 	"github.com/pkg/errors"
 	"github.com/upbound/upjet/pkg/migration"
 )
 
+const (
+	awsPackage   = "xpkg.upbound.io/upbound/provider-aws"
+	azurePackage = "xpkg.upbound.io/upbound/provider-azure"
+	gcpPackage   = "xpkg.upbound.io/upbound/provider-gcp"
+)
+
 // SSOPNames is a global map for collecting the new provider names
 var SSOPNames = map[string]struct{}{}
```

Comment:
We may get rid of this global variable by defining separate structs for the pre-processors, one for the configuration preprocessor and the other for the MR pre-processor. Then MR-imposed provider names will be collected in a separate variable than the Composition metadata imposed provider names. This will help us in the Configuration metadata converter (Please see the comment below on iterating over `SSOPNames` array in that converter.

Reply:
Done!